roles/kubernetes/preinstall/tasks/0080-system-configurations.yml

```diff
@@ -22,6 +23,26 @@
   tags:
     - bootstrap_os
 
+# SELinux might still be disabled on this host, pending reboot
+- name: Get SELinux state
+  ansible.builtin.setup:
+    gather_subset: selinux
+
+- name: SELinux configuration
+  ansible.posix.seboolean:
+    name: "{{ item }}"
+    state: true
+    persistent: true
+  loop:
+    - rsync_full_access
+  when:
+    - ansible_os_family == "RedHat"
+    - "'Amazon' not in ansible_distribution"
+    - slc.stat.exists
+    - ansible_selinux.status is defined
```

Comment:
Do we need this one ?  if the gather is it'll be there, no ?

Reply:
ansible_selinux.status might not be defined on Debian. 'Set selinux policy' needs a reboot, hence ansible_selinux.status might not be 'enabled' just yet.